### PR TITLE
CODETOOLS-7902927: JMH: Support HTML flamegraphs for async-profiler

### PR DESCRIPTION
**Pull request by shipilev**
Turns out, newer async-profiler silently eats "-o svg", expecting "-o flamegraph". It now generates HTML flamegraphs too. We need to handle both new and old flamegraph styles.

Additional testing:
 - [x] Self-built async-profiler 2.0
 - [x] Self-built async-profiler 1.8.5
 - [x] Released async-profiler 2.0
 - [x] Released async-profiler 1.8.5

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [ ] Change must be properly reviewed

### Issue
 * [CODETOOLS-7902927](https://bugs.openjdk.java.net/browse/CODETOOLS-7902927): JMH: Support HTML flamegraphs for async-profiler

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jmh pull/36/head:pull/36` \
`$ git checkout pull/36`

Update a local copy of the PR: \
`$ git checkout pull/36` \
`$ git pull https://git.openjdk.java.net/jmh pull/36/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 36`

View PR using the GUI difftool: \
`$ git pr show -t 36`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jmh/pull/36.diff">https://git.openjdk.java.net/jmh/pull/36.diff</a>

</details>

### CODE REVIEW

File: jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java [hunk 1]
```diff
@@ -341,6 +344,15 @@ private String dump(File specificOutDir, String fileNameFormatString, String con
         }
     }
 
+    static boolean svgFlamegraphs(String ver) {
+        if (ver.startsWith("1")) {
```
Comment (nevgeniev) [inline jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java:348]: well. they dropped svg @ version 1.8.5 so check is not valid :/

Comment (shipilev) [reply jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java:348]: See the discussion here: jvm-profiling-tools/async-profiler#427 -- I think Andrei made a mistake with tagging the repo, but actual 1.8.5 release should be fine.

Comment (nevgeniev) [reply jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java]: lets change it to .startsWith("1.") :)

Comment (shipilev) [reply jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java]: Right.

**Pull request merged**